codelabs/Getting_Started_with_gRPC_Java/start_here/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java

```diff
@@ -76,10 +76,11 @@ public static void main(String[] args)  {
       target = args[0];
     }
 
-        /***************************************************************
-         * Codelab Hint: create a channel using target defined above
-         ***************************************************************/
-     ManagedChannel channel = null; // Replace
+    /***************************************************************
+     * Codelab Hint: create a channel using target defined above
+      
+     ManagedChannel channel = ...
```

Comment:
The problem with this is that it causes a compile exception because `channel` isn't found